mne/decoding/tests/test_receptive_field.py

```diff
@@ -123,6 +123,12 @@ def test_receptive_field():
     rf.fit(X, y)
     assert_array_equal(rf.delays_, np.arange(tmin, tmax + 1))
 
+    # Testing fill mean
+    rf = ReceptiveField(tmin, tmax, 1, feature_names, fill_mean=True,
+                        estimator=mod)
+    rf.fit(X, y)
+    assert_array_equal(rf.delays_, np.arange(tmin, tmax + 1))
```

Comment:
This doesn't test that `fill_value` actually does anything. Can you add a test that fails in `master` but passes here?

Reply:
more over, `fill_value` is not in `ReceptiveField`'s signature [here](https://github.com/mne-tools/mne-python/blob/a22466f326cdc0724b5dbd4ae5e85005193eef14/mne/decoding/receptive_field.py#L91)

Reply:
`fill_value` isn't really meant to be public

Reply:
So this is not referring to `ReceptiveField`'s signature?
![img](http://storage7.static.itmages.com/i/17/0908/h_1504870467_2439489_76259656bd.png)

Reply:
Yeah you're right, didn't see it was in `__init__` -- it's also not supposed to be publicly exposed, so this is the wrong way to check it